src/creatwth.cpp

```diff
@@ -1019,6 +1019,11 @@ VOID CALLBACK DetourFinishHelperProcess(_In_ HWND,
         rlpDlls = NULL;
     }
 
+    // Delete the payload after execution to release the memory occupied by it
```

Comment:
I think it would be useful to clarify this comment a bit, mentioning where `s_pHelper` comes from?
Maybe something like: 

Note: s_pHelper is allocated as part of injecting the payload in `DetourCopyPayloadToProcess(..)`, it's a fake section and not data allocated by the loader.

Reply:
Maybe we should also add a comment IN `DetourCopyPayloadToProcess(..)` noting that the `VirtualAlloc` is free'd with `DetourFreePayload(..)`.

Reply:
Good idea, the latest submission has been changed in accordance with your suggestions.